aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,328 @@
+            ],
+            "responses": {
+            "200": {
+                "description": "OK",
+                "schema": {
+                "type": "array",
+                "items": {
+                    "$ref": "#/definitions/TodoItem"
+                }
+                }
+            }
+            },
+            "deprecated": false
+        },
+        "post": {
+            ...
+        }
+        },
+        "/api/Todo/{id}": {
+        "get": {
+            ...
+        },
+        "put": {
+            ...
+        },
+        "delete": {
+            ...
+    },
+    "definitions": {
+        "TodoItem": {
+        "type": "object",
+        "properties": {
+            "key": {
+            "type": "string"
+            },
+            "name": {
+            "type": "string"
+            },
+            "isComplete": {
+            "type": "boolean"
+            }
+        }
+        }
+    },
+    "securityDefinitions": {}
+    }
+
+This document is used to drive the Swagger UI which can be viewed by navigating to ``http://localhost:<random_port>/swagger/ui``
+
+.. image:: web-api-help-pages-using-swagger/_static/swagger-ui.png
+
+Each of the methods in the ToDo controller can be tested from the UI. Tap a method to expand the section, add any necessary parameters and tap "Try it out!".
+
+.. image:: web-api-help-pages-using-swagger/_static/get-try-it-out.png
+
+At this point, the basic implementation of the Swagger functionality is complete. However, there are many configuration and extensibility options available beyond the defaults. See below for more information.
+
+Customization & Extensibility 
+-----------------------------
+Swagger is not only a simple way to represent the API, but has options for documenting the object model, as well as customizing the interactive UI to match your look and feel or design language.
+
+API Info and Description
+''''''''''''''''''''''''
+For adding additional information such as the author, license, description etc.; use the ``Swashbuckle.Swagger.Model.Info`` class and use the ``.ConfigureSwaggerGen()`` method.
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Startup.cs
+    :language: c#
+    :start-after: snippet_Configure
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 16-26
+
+Then the Swagger UI produces the following more presentable information concerning the API.
+
+.. image:: web-api-help-pages-using-swagger/_static/custom-info.png
+
+XML Comments
+'''''''''''''
+In order to enable the generation of xml comments, right click the project in Visual Studio and select "Properties" and check the ``XML Documentation file`` box under the Output Settings section.
+
+.. image:: web-api-help-pages-using-swagger/_static/swagger-xml-comments.png
+
+.. note:: If you are not using Visual Studio then you can enable the "XML documentation file" flag in the ``project.json`` file by setting the ``"xmlDoc": true`` in the ``buildOptions`` section.
```

Comment:
How about delete the note
Alternatively, you can enable XML comments by setting ``"xmlDoc": true``  in *project.json*. 
```
 "buildOptions": {
    "emitEntryPoint": true,
    "preserveCompilationContext": true,
    "xmlDoc": true
  },
```